modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -207,8 +221,7 @@ With maintenance mode disabled, login and:
 * Check that the version number reflects the new installation. It’s
 visible at the bottom of your Admin page.
 * Check that your other settings are correct.
-* Go to the Apps page and review the core apps to make sure the right
-ones are enabled.
+* Go to the Apps page and review the core apps to make sure the right ones are enabled.
```

Comment:
How about using the `:experimental:` functionality here, to make the navigation that much clearer?